.github/workflows/win_cpn-64.yml

```diff
@@ -19,7 +19,7 @@ on:
       - '2.7'
       - 'main'
     paths:
-      - '.github/workflows/win_cpn-64.yml'
+      - '.github/workflows/win_cpn.yml'
       - 'companion/**'
       - 'tools/build-companion.sh'
 
```

Comment:
Why are these changes in the PR? I think this is not LR3PRO related.

Reply:
I'm also confused about this. I don't intend to modify it.

Reply:
Most likely this is because the PR is behind main, so it is effectively reverting 49c585f6d4fb40c8da04fcf0260072b466d546c1 and 48a10a35efd68a42bad74cd2e431949914002c19.